torch_uncertainty/baselines/classification/resnet.py

```diff
@@ -52,7 +52,7 @@
 
 
 # fmt: on
-class ResNet:
+class ResNet(LightningModule):
```

Comment:
Not necessary. If you're looking to have a specific type for the baseline, I advise to use Protocols. I will make a proposition on this branch.

Reply:
After discussion with @alafage, we'll just use Any and avoid Protocols for now.